lib/src/definitions.dart

```diff
@@ -263,11 +263,46 @@ class OnExternalLinkClickedResult {
   });
 }
 
+class SearchFilter {
+  /// A text that performs a search and displays only
+  /// the search results whose name or description matches the filter.
+  ///
+  /// An empty string will clear the current text filter (if any).
+  /// A null value will apply no change.
+  String? text;
+
+  /// A POI category identifier that performs a search
```

Comment:
Revisa esa frase, eu entendo algo así como "Un POI category identifier que realiza unha busca e mostra sólo...". O POI category identifier realiza a busca? Ou máis ben é usado nunha busca?
Ademais, antes era unha lista de ids, por que cambiou?